### DISCUSSION TITLE
Change the names for seller and buyer

### DISCUSSION THREAD

**Discussion by Valdnet** [baimard/gestion#170] [Ideas]
I propose to change the text strings "FROM" and "TO" to "Seller:" and "Buyer:".

![obraz](https://user-images.githubusercontent.com/47037905/154034394-31b4f1ce-6456-4968-b874-f138f368881c.png)

**Comment by les-les** [comment]
Yes would be nice and please anway you gonna handle it make it look like This 
FROM 
Selling Stuff Over The Internet llc.
www-roada
CA-0911

TO 
Buyin all stuff 


There must be 
A hard Break after FROM and TO 
because this is not part of the Comany Name 

a bit of a tricky issue if someone not wanna pay your bill
